### ISSUE TITLE
Question, how to run without camera

### ISSUE THREAD

**Issue by ecward** [kujason/avod#24]
Hi!

Thank you for sharing your excellent work. In the paper there is a row in Table III where you use BEV only features (RPN BEV Only).  I am interested in using this network, is there a configuration file available for this?

Also, would it be possible for you to share the trained models?

**Comment by melfm** [comment]
We did not make this configurable however modifying the RPN to use BEV only is fairly straightforward. Inside the `rpn_model.py`, you need to disable image and BEV fusion (see code [proposal_roi_fusion](https://github.com/kujason/avod/blob/master/avod/core/models/rpn_model.py#L351) and instead set the `rpn_fusion_out` to `bev_proposal_rois` features directly.  You can then commend out the generation of `img_proposal_rois` as it won't be needed.

With regards to sharing trained models, since the network trains relatively fast and there is no pre-training required, we decided not to publish the trained models.

Hope this helps.

**Comment by melfm** [comment]
You could try using the BEV features only for second stage too, however the performance is expected to drop (specially for detecting smaller objects like pedestrian and cyclists).

**Comment by peiyunh** [comment]
@melfm Thanks for the answer. I was wondering if we can do this by setting `path_drop_probabilities` to [0, 1].

**Comment by melfm** [comment]
Yeah you could also disable it via `path_drop`.

**Comment by peiyunh** [comment]
@melfm That makes sense. It would involve more changes and it is not really computationally efficient neither. Thanks for the clarification.

**Comment by LouieYang** [comment]
@melfm Thanks for the answer. I was wondering if there is any camera-only detection result on Kitti that we can refer to make sure whether our modification is correct?

**Comment by asharakeh** [comment]
@LouieYang please check our paper for BEV only results on the validation set.

**Comment by Bovey0809** [comment]
@LouieYang Sorry for bothering you, can I ask for the BEV only version.
I'm trying to modify the code but It seems too complicated for me.

**Comment by Tgaaly** [comment]
I'm using the KITTI cpp code (evaluate_object_3d_offline.cpp) to evaluate my birds eye view (BEV) bounding box detections. The code filters the detections into easy, moderate and hard by the height of the detected object's bounding box in the image space (i.e. pixels). I do not have image detections at all and so all my detections are being ignored (lines 447-452). I'm experimenting solely on BEV data. Any idea how to evaluate ONLY BEV detections (without any corresponding image detections). Is there a reference somewhere for this?

**Comment by lukui1995** [comment]
@LouieYang   Sorry for bothering you, can I ask for the BEV only version.
